### PR TITLE
upgrade the different graphql web interfaces and dependencies

### PR DESCRIPTION
**Pull request by mayel**

### CODE REVIEW

File: lib/absinthe/plug/graphiql/graphiql_playground.html.eex [hunk 1]
```diff
@@ -55,7 +56,19 @@ add "&raw" to the end of the URL within a browser.
     </div>
   </div>
   <script>
-    var options = {};
+    var options = { // FIXME: looks like these options are ignored
```
Comment (benwilson512) [inline lib/absinthe/plug/graphiql/graphiql_playground.html.eex:59]: Can you elaborate on this?

File: lib/absinthe/plug/graphiql/assets.ex [hunk 2]
```diff
@@ -35,38 +45,46 @@ defmodule Absinthe.Plug.GraphiQL.Assets do
...
-    {"@absinthe/graphql-playground", "1.2.0",
-     [
-       {"build/static/css/middleware.css", "playground.css"},
-       {"build/static/js/middleware.js", "playground.js"}
-     ]},
-    {"@absinthe/socket-graphiql", "0.1.1",
+    {
+      "graphql-playground-react",
+      "1.7.23",
+      # should probably be deprecated eventually, as playground is being shut down in favour of GraphiQL: https://github.com/prisma-labs/graphql-playground/issues/1143
```
Comment (benwilson512) [inline lib/absinthe/plug/graphiql/assets.ex:81]: I think we should honestly deprecate these now and begin the process of extracting _all_ of the graphiql interfaces into dedicated hex deps. The plan with Absinthe.Plug 2.0 will be to not have any built in.

Comment (mayel) [reply lib/absinthe/plug/graphiql/assets.ex:81]: You plan sounds good. Some people might want a Playground hex option then. Workspace on the other hand is unmaintained but very useful (it makes it easy to add custom headers). So until the next major version of GraphiQL comes out (with pluginisation) both of these are handy...

Comment (benwilson512) [reply lib/absinthe/plug/graphiql/assets.ex:81]: For sure.

**Comment by benwilson512** [comment]
Have you tested these updates with subscriptions by the way?

**Comment by mayel** [comment]
Ah no, as I'm not yet using subscriptions

**Comment by binaryseed** [comment]
Closing this particular ticket since our path forward is to remove GraphiQL from `Absinthe.Plug`...

**Pull request closed without merging**